.github/workflows/build-container-images.yaml

```diff
@@ -71,3 +75,25 @@ jobs:
           tags: ghcr.io/rackerlabs/understack/argo-ironic-client-python3.11.8:latest,ghcr.io/rackerlabs/understack/argo-ironic-client-python3.11.8:${{ env.VERSION_PYTHON_IRONIC }}
           labels: |
               org.opencontainers.image.version=${{ env.VERSION_PYTHON_IRONIC }}
+
+      - name: Build and deploy Argo Utils image
+        uses: docker/build-push-action@v5
+        with:
+          context: argo-workflows/generic/
+          file: argo-workflows/generic/containers/Dockerfile.argo_utils
+          # push for all main branch commits
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ghcr.io/rackerlabs/understack/argo-utils-python3.11.8:latest,ghcr.io/rackerlabs/understack/argo-utils-python3.11.8:${{ env.VERSION_ARGO_UTILS }}
```

Comment:
Please have a look at a [discussion around including python version in non-base containers](https://github.com/rackerlabs/understack/pull/78#discussion_r1591640481) and let us know what you think. If you agree, I'll create separate PR to adjust the `ironic-nautobot-sync` image name.